dev/tools/merge-pr.sh

```diff
@@ -1,50 +1,134 @@
 #!/usr/bin/env bash
 
 set -e
+set -o pipefail
+
+API=https://api.github.com/repos/coq/coq
 
 # This script depends (at least) on git and jq.
 # It should be used like this: dev/tools/merge-pr.sh /PR number/
 
-#TODO: check arguments and show usage if relevant
+RED="\033[31m"
+BLACK="\033[30m"
+GREEN="\033[32m"
+BLUE="\033[34m"
+YELLOW="\033[33m"
+info() {
+  echo -e "${GREEN}info:${BLACK} $@ ${BLACK}"
```

Comment:
Do not use $@ in a string, instead use $*.

Reply:
why shouldn't I?
```
gares@gargamel:~/COQ/coq$ cat a.sh 
foo(){
  echo "foo $@ bar"
}
foo bla bla "bla bla"
gares@gargamel:~/COQ/coq$ bash a.sh 
foo bla bla bla bla bar
```

Reply:
Because it makes shellcheck complain https://github.com/koalaman/shellcheck/wiki/SC2145

Reply:
No, this is the white space linter syndrome again.

Tell me what is wrong, not that a linter complains.
If there is a reason I'm fine making a fix. According to the doc of bash, my code is just fine.

Reply:
Alternatively, because if someone ever decides to get rid of the `${GREEN}info:${BLACK} ` prefix, and leaves `$@`, then passing something like `-n foo` to this function will result in `echo` printing without a newline:
```bash
jgross@jgross-Leopard-WS:~/Documents/repos/fiat-crypto$ cat a.sh
foo(){
    echo "$@ bar"
}
foo -n bla bla "bla bla"
jgross@jgross-Leopard-WS:~/Documents/repos/fiat-crypto$ bash a.sh
bla bla bla bla barjgross@jgross-Leopard-WS:~/Documents/repos/fiat-crypto$
```